edudoc/delegate-crash-course/delegate_crash_course.md

```diff
@@ -65,11 +65,11 @@ To request an official competition you first need to create a new competition by
 ### Some general rules about requesting a competition:
 
 - The competition must be announced **at least 28 days** before the scheduled start of the competition. You should **submit the competition earlier than this** as a safety measure so that if there is an issue with the competition, then the competition will likely still be announced before the 28-day deadline.
-- The competition must comply with the [Competition Requirements Policy](https://www.worldcubeassociation.org/documents/policies/external/Competition%20Requirements.pdf). This means your competition should be at least 10 kilometers and 5 days away from all other competitions. There can be exceptions made to the Proximity Policy if the Delegate has valid arguments. Examples can be found in the policy.
+- The competition must comply with the [Competition Requirements Policy](https://documents.worldcubeassociation.org/documents/policies/external/Competition%20Requirements.pdf). This means your competition should be at least 10 kilometers and 5 days away from all other competitions. There can be exceptions made to the Proximity Policy if the Delegate has valid arguments. Examples can be found in the policy.
```

Comment:
This would be better as wcadoc{} too

Reply:
Same for the rest of the changes in this doc

Reply:
I didn't change those to wcadoc because we are going to replace the entire document with the handbook.

Do you still want to change it?

Reply:
Well, I thought since we're making a change, might as well use the shortcut, but I guess it doesn't matter